### PR TITLE
Localstorage

### PR DESCRIPTION
**Pull request by robintenhoven**
- Interests are now in a scrollview that scales based on the screen width and height through Dimensions.
- The TouchableOpacity, where the view is wrapped inside, is now clickable.
- Upon clicking, saves an empty array in localstorage or removes if it already contains the clicked interest.
- Imagepath is implemented into the jsonfile for a dynamic require for dynamically generated images.
- A border is given when the interest is selected and border is removed by deselecting.
- The TouchableOpacity from Interests is converted into a TouchableInterest component.
- Implemented the StorageHelper

### CODE REVIEW

File: components/features/TouchableInterest.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	},
+	interestSelected: {
+		marginLeft: 10,
+		marginTop: 10,
+		width: 100,
+		height: 100,
+		borderRadius: 10,
+		borderWidth: 3,
+		borderColor: '#d6d7da',
+		backgroundColor: '#FF6060'
```
Comment (Mike-U5) [inline components/features/TouchableInterest.tsx]: Use Colors.BgDark

File: components/features/Interests.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		}
+}
+
+const styles = StyleSheet.create({
+	main: {
+		flex: 1
+	},
+	container: {
+		flex: 1,
+		backgroundColor: '#fff',
```
Comment (Mike-U5) [inline components/features/Interests.tsx]: Use Colors.White

File: components/features/TouchableInterest.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+			});
+		}
+}
+const styles = StyleSheet.create({
+	interestContainer: {
+		marginLeft: 10,
+		marginTop: 10,
+		width: 100,
+		height: 100,
+		backgroundColor: '#FF6060'
```
Comment (Mike-U5) [inline components/features/TouchableInterest.tsx]: Use Colors.bgDark

File: components/features/TouchableInterest.tsx (hunk 1 above)
Comment (robintenhoven) [reply components/features/TouchableInterest.tsx]: Thank you, 優しい友達

**Pull request merged**